ddtrace/tracer/span.go

```diff
@@ -158,6 +151,26 @@ func (s *span) setTagString(key, v string) {
 	}
 }
 
+// setTagBool sets a boolean tag on the span.
+func (s *span) setTagBool(key string, v bool) {
+	switch key {
+	case ext.AnalyticsEvent:
+		if v {
+			s.setTagNumeric(ext.EventSampleRate, 1.0)
+		} else {
+			s.setTagNumeric(ext.EventSampleRate, 0.0)
+		}
+	case ext.ManualDrop:
```

Comment:
Is the tag supposed to be added to the span in addition to adjusting sampling priority's value?

Reply:
It's just a short-hand alias to simplify setting sampling priority, so no.